content/python/concepts/built-in-functions/terms/setattr/setattr.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Setattr()'
+Description: 'Sets the value of the attribute of an object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`Setattr()`** function is a built-in Python function used to set the value of a named attribute of an object. It allows you to dynamically assign or modify attributes of an object at runtime.
```

Comment:
[Proofreading/Nitpick] See my feedback on [line 50](https://github.com/Codecademy/docs/pull/2467/files#r1225934612) for context. 
```suggestion
The **`Setattr()`** function is a built-in Python function used to set the value of a named attribute of an object. It allows a developer to dynamically assign or modify attributes of an object at runtime.
```